javascript-view-swapping/main.js

```diff
@@ -16,7 +16,7 @@ tabContainer.addEventListener('click', function (e) {
     const dataView = e.target.getAttribute('data-view');
     for (let i = 0; i < allView.length; i++) {
       if (allView[i].getAttribute('data-view') === dataView) {
-        allView[i].className = dataView;
+        allView[i].className = 'view';
       } else {
         allView[i].className = 'hidden';
```

Comment:
This will need to be `view hidden` so that you don't lose the initial class. `className` removes all classes an element already has, so we need to make sure we keep it.